test/integration/runner_task_test.rb

```diff
@@ -238,7 +221,7 @@ def test_run_fails_if_task_template_is_blank
     ], in_order: true)
   end
 
-  def test_run_bang_fails_if_task_template_or_args_are_invalid
+  def test_run_bang_fails_if_task_template_is_invalid
```

Comment:
I think this test can be removed now, as this is already tested in the method above.

Reply:
Nevermind, one is `run` and the other `run!`.